### ISSUE TITLE
[BUG] - Search results are not shown after errors fixes in customized query

### ISSUE THREAD

**Issue by banothurameshnaik** [cars10/elasticvue#68]
**Description**
In the search, if any syntax errors are there and after correcting syntax errors also matched search results are not showing up.

**Steps To Reproduce**
1. Open elasticvue
2. Click on SEARCH tab
3. Click on Customize Query
4. Search some index which has some documents
4. Documents will be shown once we hit search button
Example: Search
```json
{
	"query": {
		"query_string": {
			"query": "*"
		}
	},
	"size": 10,
	"from": 0,
	"sort": []
}
```
5. Now include some syntax errors in the customized query
Example: `I removed comma after "size": 10` in the query to make it syntax error
```json
{
	"query": {
		"query_string": {
			"query": "*"
		}
	},
	"size": 10
	"from": 0,
	"sort": []
}
```
6. On hitting Search button, we will get `invalid search query` message
7. Now correct the syntax of query and hit the search button
```json
{
	"query": {
		"query_string": {
			"query": "*"
		}
	},
	"size": 10,
	"from": 0,
	"sort": []
}
```
7. Now no matched documents are shown even though matched documents exists.

**Screenshots**
![before-documents](https://user-images.githubusercontent.com/25602365/132326278-36787c01-89fd-4eca-8850-13bd0571d4c2.png)
![invalid-query](https://user-images.githubusercontent.com/25602365/132326312-3296f568-991b-4686-9b6b-865e4dc788b3.png)
![no-documents-after-correcting-errors](https://user-images.githubusercontent.com/25602365/132326321-76383399-de37-4f64-801e-4b400bd4b1a6.png)

**Environment (please include the following information):**
* Elasticsearch version: `7.4.2`
* Operating system: `Mac OS Big Sur` 
* Browser + version: `Google Chrome, Version 93.0.4577.63` 
* Elasticvue version: `Elasticvue 0.36.3`
* How are you running elasticvue? (docker, browser extension, web app):

**Additional context**
Video of recording issue can be found here 
https://www.awesomescreenshot.com/video/5131237?key=b8c3c6c7590453cd3e4401c23505c61f

**Comment by cars10** [comment]
Thanks for the detailed bug report! I can reproduce this and will look into it.

**Comment by cars10** [comment]
This will be fixed in  the next version, `0.36.4`.
